lightning_pose/models/heatmap_tracker.py

```diff
@@ -133,16 +134,31 @@ def run_subpixelmaxima(
                 - confidences of shape (batch, num_keypoints)
 
         """
+
```

Comment:
if you have some energy left, you could, above the `HeatmapTracker`, copy and paste this into a `SubpixelMaxima` class. `self.upsample()` is one method, offsets can be made into `@property`.
the class could be initialized in `HeatmapTracker.__init__()` and its forward method can be called as `HeatmapTracker.subpixel_maxima(inputs)`.
Not urgent, but seems mostly copy and pasting and will lighten up the model script. up to you, looks good.